src/common/checkbox/Checkbox.tsx

```diff
@@ -4,23 +4,37 @@ import classNames from 'classnames';
 import styles from './checkbox.module.scss';
 import Icon from '../icon/Icon';
```

Comment:
Can we switch to Icon component from HDS?

Reply:
The [Icon component](https://city-of-helsinki.github.io/helsinki-design-system/storybook/?path=/story/icons--iconcheck) from HDS won't work for this checkbox, but I can replace the Icon here with an imported SVG, so the checkbox would be a self-contained component as we discussed.
What do you think?